core_perception/ndt_gpu_ros2/config/README.md

```diff
@@ -0,0 +1 @@
+Config folder is currently required for building
```

Comment:
Do we actually need config folder? Can we just remove it from the CMakeLists.txt?
Line 116. Same for CPU package